src/Vimus/Run.hs

```diff
@@ -193,18 +195,27 @@ run host port ignoreVimusrc = do
         -- load default mappings
         Command.runCommand "runtime default-mappings"
 
-        -- source ~/.vimusrc
+        configHome <- liftIO $ getEnv "XDG_CONFIG_HOME"
```

Comment:
I don't feel too strongly about it but this could do with some golfing to make it a little more succinct/easier to follow. Like: take a list of alternatives and try each in order.

Reply:
also sorry for the late responses but july was a heavy month for me